frontend/src/app/lib/utils.js

```diff
@@ -1,5 +1,8 @@
 import querystring from 'querystring';
 
+import { experimentL10nId, l10nId, l10nIdFormat, lookup } from '../../../tasks/util';
```

Comment:
In order to reuse this logic in the gulp tasks, I had to move these to their own module that doesn't require babelification. I imported them here for testing and other purposes.